{
  "schema_version": "1.6.3",
  "id": "CVE-2021-22201",
  "details": "An issue has been discovered in GitLab CE/EE affecting all versions starting from 13.9. A specially crafted import file could read files on the server.",
  "aliases": [],
  "related": [],
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:N/A:N"
    }
  ],
  "published": "2021-04-02T17:15:00Z",
  "modified": "2021-04-02T17:15:00Z",
  "affected": [
    {
      "package": {
        "ecosystem": "Ubuntu:Pro:16.04:LTS",
        "name": "gitlab"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            }
          ]
        }
      ],
      "versions": [
        "8.4.3+dfsg-9",
        "8.4.3+dfsg-12",
        "8.5.8+dfsg-5"
      ]
    }
  ],
  "references": [
    {
      "type": "REPORT",
      "url": "https://ubuntu.com/security/CVE-2021-22201"
    },
    {
      "type": "REPORT",
      "url": "https://gitlab.com/gitlab-org/cves/-/blob/master/2021/CVE-2021-22201.json"
    },
    {
      "type": "REPORT",
      "url": "https://gitlab.com/gitlab-org/gitlab/-/issues/325562"
    },
    {
      "type": "REPORT",
      "url": "https://hackerone.com/reports/1132378"
    },
    {
      "type": "REPORT",
      "url": "https://www.cve.org/CVERecord?id=CVE-2021-22201"
    }
  ]
}